docker-app/qfieldcloud/core/models.py

```diff
@@ -1415,7 +1421,65 @@ class Meta:
         managed = False
 
 
-class Delta(models.Model):
+class AbstractValidatedProjectOwner(models.Model):
+    """ "
+    Assumes a foreign key field 'project' for a Project
+    """
+
+    class Meta:
+        abstract = True
+
+    @property
+    def account(self):
+        return self.project.owner.useraccount
+
```

Comment:
```suggestion
```
When we extend this as a mixin in `Job` or `Delta`, it means there will be `job.account` and `delta.account`, which will be a bit misleading. I would propose we remove this one.